databases-analytics/databases/hello-pg-java/README.md

```diff
@@ -0,0 +1,57 @@
+## Deploy a Quarkus application to access to DBaaS
+
+### General information
+ - 🔗 [Getting started with Kubernetes database operator](https://help.ovhcloud.com/csm/en-gb-public-cloud-databases-database-operator?id=kb_article_view&sysparm_article=KB0058469)
+ - 🔗 [Using the OVHcloud Managed Kubernetes LoadBalancer](https://help.ovhcloud.com/csm/en-gb-public-cloud-kubernetes-using-lb?id=kb_article_view&sysparm_article=KB0050007)
+ - 🔗 [OVH token generation page](https://www.ovh.com/auth/api/createToken?GET=/*&POST=/*&PUT=/*&DELETE=/*)
+ - 🔗 [Quarkus](https://quarkus.io/)
+
+### Set up
+  - A JDK (at least 21)
```

Comment:
```suggestion
  - A JDK (at least 21)
```

You need also `helm` and `kubectl` CLI installed :)

Reply:
fixed with https://github.com/ovh/public-cloud-examples/pull/39/commits/86151f4b7586557c607445909c1c630d67093480